js/hw11.js

```diff
@@ -0,0 +1,136 @@
+// Знайти суму та кількість позитивних елементів.
+const arr1 = [16, -37, 54, -4, 72, -56, 47, 4, -16, 25, -37, 46, 4, -51, 27, -63, 4, -54, 76, -4, 12, -35, 4, 47];
+let sum1 = 0;
+for (let i = 0; i = arr1.length; i += 1) {
+    const anElement = arr1.shift();
+    if (anElement > 0) {
+        sum1 += Number(anElement);
+    }
+}
+console.log(sum1);
+
+// Знайти мінімальний елемент масиву та його порядковий номер.
+
+const arr2 = [16, -37, 54, -4, 72, -56, 47, 4, -16, 25, -37, 46, 4, -51, 27, -63, 4, -54, 76, -4, 12, -35, 4, 47];
+const minElement = [arr2[0]];
+for (let i = 0; i = arr2.length; i += 1) {
+    const anElement = arr2.shift();
+    if (anElement < minElement) {
+        minElement.shift();
+        minElement.push(anElement);
+    }
+}
+console.log(minElement);
+
+const arr2_1 = [16, -37, 54, -4, 72, -56, 47, 4, -16, 25, -37, 46, 4, -51, 27, -63, 4, -54, 76, -4, 12, -35, 4, 47];
+console.log(arr2_1.indexOf(Number(minElement.join())));
```

Comment:
Код буде працювати, але мені не зовсім зрозуміло
- навіщо виеористовувати масив для зберігання мінімального елемента
- навіщо використовувати метод `shift`
- навіщо змінювати масив з вихідними даним

```suggestion
const minElement = arr2[0];
for (let i = 0; i = arr2.length; i += 1) {
    if (arr[i] < minElement) {
        minElement = arr[i]
    }
}
console.log(minElement);

const arr2_1 = [16, -37, 54, -4, 72, -56, 47, 4, -16, 25, -37, 46, 4, -51, 27, -63, 4, -54, 76, -4, 12, -35, 4, 47];
console.log(arr2_1.indexOf(minElement));
```